Partition (WBP)/Functions/VI0033C.VTI0033.fun.yml

```diff
@@ -172,75 +160,39 @@
     Define_Map For(*BOTH) Class(#BIFA002A) Name(#this_rc)
     
     * Create Test Data list
-    
+    Clr_List Named(#w_lOriginal)
     * Real Fields: #BIFP0300A #BIFA256A #BIFA010A #BIFA010C #BIFA010D #BIFP0702A #BIFN0702A
     #LF_i := 1
     
     #BIFP0300A := #LF_i
     #LF_i += 1
-    #BIFA256A := 'Insert'
-    #BIFA010A := 'AAAAAAAAAA'
-    #BIFA010C := 'CCCCC'
-    #BIFA010D := 'D'
-    #BIFA010B := #BIFA010A
-    #BIFA010E := #BIFA010C
-    #BIFA010F := #BIFA010D
-    #BIFP0702A := 99999.8
-    #BIFN0702A := #BIFP0702A
-    #BIFN0702B := #BIFN0702A
-    Use Builtin(CONCAT) With_Args(#BIFA010A #BIFA010C #BIFA010D) To_Get(#BIFA040A)
-    Use Builtin(CONCAT) With_Args(#BIFA010A #Com_Owner.mthSigned2String( #this_rc #BIFN0702A ) #BIFA010C) To_Get(#BIFA040B)
+    #RDDMMYYYY := '21122023'
```

Comment:
I explicitly stated what MM, DD and YY would be so that you set those and then had a standard algorithm for setting the value of each Field based on these base parts.
So
```
#MM := 12
#DD :=21
#YY := 23
#RDDMMYYYY := #DD + #MM + '20' + #YY /* Put these 2 lines in a method along with setting all the other Columns in the table */ 
#VMMDDYYYY := #MM + #DD + '20' + #YY /* And call that method instead. */
```